src/main/java/com/edhou/songr/MvgConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.edhou.songr;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class MvgConfig implements WebMvcConfigurer {
+
```

Comment:
This file exposes the `uploaded-files` directory to HTTP requests at `/uploaded-photos` path